src/objdump.py

```diff
@@ -150,6 +155,81 @@ def parse_relocated_line(line: str) -> Tuple[str, str, str]:
     return before, imm, after
 
 
+def process_reloc(reloc_row: str, prev: str):
```

Comment:
return type; I think `mypy` will complain as is, please check

Reply:
ohh, I thought black was auto adding return types... but yea I need to install, mypy. I'll do that and add return type